README.md

```diff
@@ -24,4 +24,4 @@ A customizable vuejs tree viewer
 [![Edit vue3-tree](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/boring-leaf-v7b2s?file=/src/App.vue)
 
 #### How to use
-You can review the [document](https://teamseodo.github.io/vue3-tree/) for the use of the library.
+You can review the [document](https://wopehq.github.io/vue3-tree/) for the use of the library.
```

Comment:
```suggestion
You can review the [document](https://wopehq.github.io/vue3-tree/guide/#installation) for the use of the library.
```

Reply:
That's a good suggestion. I will update it.